config/Config-kernel.in

```diff
@@ -1054,6 +1054,13 @@ if KERNEL_IP_MROUTE
 
 endif
 
+config KERNEL_MPTCP
+	bool "Enable IPv4 MultiPath TCP support"
+	default y
```

Comment:
default y? How much extra code will it add to the image?

Reply:
already fixed.